src/main/java/gov/nih/nci/evs/api/controller/ErrorHandlerController.java

```diff
@@ -126,7 +126,8 @@ protected Map<String, Object> getErrorAttributes(
               .toString()
               .split("\n")[0]
               .replaceFirst(".*?\"", "")
-              .replaceFirst("[\"\r]+$", "");
+              .replaceFirst("[\"\r]+$", "")
+              .replaceFirst("400 BAD_REQUEST \"", "");
```

Comment:
Do we have a way to identify a 400 error so that replacing it with an empty string shows the user they made a bad sparql query?

Reply:
This whole "trace" thing is a hack that we'll fix when we upgrade to Spring 3.  it was just producing a weird error message in a certain case and this was a cheap way of fixing it (for now).  NOT a generalizeable solution :)